app/presenters/summary/sections/client_details.rb

```diff
@@ -30,7 +30,7 @@ def answers
 
           Components::FreeTextAnswer.new(
             :nino, applicant.nino,
-            change_path: edit_steps_client_has_nino_path
+            change_path: edit_steps_client_has_nino_path, show: true
```

Comment:
As NINO is now sometimes optional, we want to always show this row, and populate it with None if nothing was entered